cmd/snap/main.go

```diff
@@ -129,11 +129,19 @@ func init() {
 	}
 }
 
+var accessDeniedHelp = `Access denied.
+
+Please use "sudo" or the "snap login <userid>" command
```

Comment:
Can I join the bikeshed? :-)

`Access denied (snap login --help).`

Then, let's fix snap login to be more clear:

```
type cmdLogin struct {
        Positional struct {
                // FIXME: add support for translated descriptions
                //        (see cmd/snappy/common.go:addOptionDescription)
                UserName string `positional-arg-name:"email" description:"login.ubuntu.com email to login as"`
        } `positional-args:"yes" required:"yes"`
}

var shortLoginHelp = i18n.G("Authenticates on snapd and the store")

var longLoginHelp = i18n.G(`
The login command authenticates on snapd and the snap store and saves credentials
into the ~/.snap/auth.json file. Further communication with snapd will then be made
using those credentials.

Accounts may be created at https://login.ubuntu.com.
`)
```